src/template.handlebars

```diff
@@ -4,6 +4,13 @@
     <test-suite type="Assembly" name="{{name}}" start-time="{{startTime}}" end-time="{{endTime}}">
         {{#testCases}}
         <test-case name="{{name}}" fullname="{{../name}} - {{name}}" result="{{result}}" duration="{{duration}}">
+            {{#if hasMetaData}}
+            <properties>
+                {{#each meta}}
+                    <property name="{{@key}}" value="{{this}}" />
```

Comment:
Removing 4 spaces before <property name=...> would make the report.xml a little look better. But ok as is.